src/registry.rs

```diff
@@ -467,14 +492,23 @@ impl<'reg> Registry<'reg> {
         self.decorators.get(name).map(|v| v.as_ref())
     }
 
-    /// Return all templates registered
-    pub fn get_templates(&self) -> &HashMap<String, Template> {
-        &self.templates
+    /// Return all  registered template names
+    pub fn get_template_names(&self) -> Vec<String> {
```

Comment:
Sad that we didn't catch the 4.0 release train to ship this break change.

With the `Source` API, the original `get_templates() -> &HashMap<String, Template>` has lost its meaning